light-client/src/client.rs

```diff
@@ -177,6 +179,31 @@ impl InnerLightClient {
             return Err(Error::UnexpectedRevisionHeight(height.revision_height()));
         }
 
+        if client_state.fork_specs.is_empty() {
+            return Err(Error::EmptyForkSpec);
+        }
+
+        verify_sorted_asc(&client_state.fork_specs)?;
+
+        for spec in &client_state.fork_specs {
```

Comment:
Although the `fork_specs` likely has few elements, since it's already partially sorted, if the first elements of both height and timestamp are at least the MINIMUM value, further checks seem unnecessary.

Reply:
Done by [4578dc7](https://github.com/datachainlab/parlia-elc/pull/83/commits/4578dc7341bd6efe1ca974dbf697d515609e93c0)